docs/Configuring_an_Application_Server.md

```diff
@@ -122,3 +122,32 @@ To update the configuration, invoke `curl` as follows.  The first stage builds t
     } | curl -X PUT http://$hs_hostname:8889/irs/<irs-uuid>/service_profiles/<service-profile-uuid>/filter_criteria --data-binary @-
 
 The subscriber will now have the desired configuration. You can confirm this by running the retrieval command again.
+
+## SIP-over-UDP configuration
+
+While it is recommended that you configure your deployment to send SIP over TCP, it is possible to create a SIP-over-UDP deployment, with certain restrictions.
+
+### Restrictions
+
+If you want to create a SIP-over-UDP deployment, it will be necessary for all of your IMS Core's SIP peers to send SIP over UDP to the IMS Core. It is not possible to set up some peers to use TCP and some to use UDP.
+
+### Configuration
+
+To enable SIP-over-UDP, you will need to set the following configuration options.
+
+In `/etc/clearwater/shared_config` set or update the field:
+
+    scscf_uri=sip:scscf.<sprout_hostname>;transport=udp
+
+In `/etc/clearwater/local_config` set or update the field on each of your Sprout nodes:
+
+    scscf_node_uri=sip:<scscf_ip_address>:5054;transport=udp
```

Comment:
Maybe `local_ip` is better than `scscf_ip_address` (as there will already be a local_ip variable in local_config, so it'll be clear what the value should be?).